### PR TITLE
all defaults must be ...

### PR DESCRIPTION
**Pull request by JelleZijlstra**
This is in the typeshed style guide and I fixed (almost) all offending code. We should
put it in flake8-pyi so I don't have to enforce it in code reviews in the future.

### CODE REVIEW

File: pyi.py [hunk 1]
```diff
@@ -354,6 +366,7 @@ def should_warn(self, code):
 Y008 = 'Y008 Unrecognized platform "{platform}"'
 Y009 = 'Y009 Empty body should contain "...", not "pass"'
 Y010 = 'Y010 Function body must contain only "..."'
+Y011 = 'Y011 Default values for typed arguments must be "..."'
```
Comment (ambv) [inline pyi.py]: Put it also in the README.

Comment (JelleZijlstra) [reply pyi.py]: Done. Also rebased the branch because I made it on top of an old commit before we added docs to README.

**Comment by JelleZijlstra** [comment]
The CI value is expected for now until python/typeshed#1998 and python/typeshed#1999 are merged.

**Comment by ambv** [comment]
Oh. Merged.

**Comment by ambv** [comment]
Hm, there's some PyPI issue now. I uploaded a broken 18.3.0 release so uploaded a fixed 18.3.1 right after. Yet, pip install flake8-pyi claims 18.3.0 is the newest, even though I can see 18.3.1 on both pypi.python.org and pypi.org.

Are you able to install 18.3.1?

**Comment by ambv** [comment]
http://pypi.org/p/flake8-pyi and https://pypi.python.org/pypi/flake8-pyi both list 18.3.1 as available.

**Comment by JelleZijlstra** [comment]
Thanks for doing the release! 

I just ran `pip install flake8-pyi` and got 18.3.1.

```
$ pip install flake8-pyi
Collecting flake8-pyi
  Downloading flake8_pyi-18.3.1-py36-none-any.whl
Requirement already satisfied: flake8>=3.2.1 in ./.venv/lib/python3.6/site-packages (from flake8-pyi)
Requirement already satisfied: attrs in ./.venv/lib/python3.6/site-packages (from flake8-pyi)
Requirement already satisfied: pycodestyle<2.4.0,>=2.0.0 in ./.venv/lib/python3.6/site-packages (from flake8>=3.2.1->flake8-pyi)
Requirement already satisfied: pyflakes<1.7.0,>=1.5.0 in ./.venv/lib/python3.6/site-packages (from flake8>=3.2.1->flake8-pyi)
Requirement already satisfied: mccabe<0.7.0,>=0.6.0 in ./.venv/lib/python3.6/site-packages (from flake8>=3.2.1->flake8-pyi)
Installing collected packages: flake8-pyi
Successfully installed flake8-pyi-18.3.1
```
Maybe you have some local cache?

**Comment by ambv** [comment]
I guess so. Will tinker after dinner.

**Pull request merged**